### PR TITLE
Show user session

### PR DESCRIPTION
**Pull request by pkarman**
# Description
Adds a new route `/test/users/me` that will dump the current user session for debugging purposes.

In addition, closes a security/performance issue where all the User records were being written to the response on the `/test/users` page in a production environment.

### CODE REVIEW

File: spec/feature/log_in_as_user_spec.rb [hunk 1]
```diff
@@ -19,19 +19,37 @@
...
-    safe_click("#button-Log-in-as-user")
-    expect(page).to have_content("ANNE MERICA (DSUSER)")
-    expect(page).to have_content("Certification Help")
-    visit "test/users"
-    expect(page).not_to have_content("Log in as user")
-    find("a", text: "ANNE MERICA (DSUSER)").click
-    find("a", text: "Sign Out").click
-    expect(page).to have_content("DSUSER")
-    expect(page).to have_content("admin page")
+  context "user is Global Admin" do
```
Comment (pkarman) [inline spec/feature/log_in_as_user_spec.rb]: this first test is just a refactor to use `before/after` blocks.

File: app/controllers/test/users_controller.rb [hunk 2]
```diff
@@ -158,6 +157,30 @@ def show_error
 
   private
 
+  def user_session
+    (params[:id] == "me") ? session : nil
```
Comment (pkarman) [inline app/controllers/test/users_controller.rb]: we can't see other users' sessions at this point but might in future allow for viewing a user's record or other data.

File: spec/feature/log_in_as_user_spec.rb [hunk 3]
```diff
@@ -19,19 +19,37 @@
...
+    before do
+      Functions.grant!("Global Admin", users: ["DSUSER"])
+    end
+
+    after do
+      Functions.client.del("Global Admin")
+    end
+
+    scenario "user is able to log in as user" do
```
Comment (leikkisa) [inline spec/feature/log_in_as_user_spec.rb]: Should we stub "dependencies_faked?" here to make sure this still works in prod?

Comment (pkarman) [reply spec/feature/log_in_as_user_spec.rb]: good call. I will open another PR

**Pull request merged**